server/images.go

```diff
@@ -366,10 +375,11 @@ func CreateModel(ctx context.Context, name string, path string, fn func(resp api
 					return err
 				}
 
-				// copie the model metadata
+				// copy the model metadata
 				config.ModelFamily = source.ModelFamily
 				config.ModelType = source.ModelType
 				config.FileType = source.FileType
+				config.Capabilities = source.Capabilities
```

Comment:
`Capabilities` is a weird name for this field. `RequireCapabilities` or `NeedCapabilities` is probably better to communicate these are needed to run correctly